### DISCUSSION TITLE
When will Bun be available in windows?

### DISCUSSION THREAD

**Discussion by moraby256** [oven-sh/bun#4363] [Q&A]
Is there any intention to use **Bun in Windows System** in the Future Versions of Bun?

and i don't mean by that **WSL**.

**Comment by xhyrom** [comment]
Yes, we also have tracking issue https://github.com/oven-sh/bun/issues/43 and pull request https://github.com/oven-sh/bun/pull/4410

We don't have ETA, just need to wait. Don't want to rush things again.

**Reply by tomerh2001** [reply]
Again?
